<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewItemDialog</class>
 <widget class="QDialog" name="NewItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Item</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="goodNameInput">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>73</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Good Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceInput">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>113</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit-Price</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityInput">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>147</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhFormattedNumbersOnly</set>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goodPickerButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scanButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
